src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt

```diff
@@ -67,9 +67,11 @@ class EvolvePokemon : Task {
                 if (settings.evolveBeforeTransfer.contains(it.pokemonId)) {
                     Log.yellow("Evolving ${it.pokemonId.name} before transfer")
                     val evolveResult = it.evolve()
```

Comment:
I gave this a whirl. I had to add a timeout before the it.evolve() call. Seemed to work fine then.